src/test/kotlin/lotto/model/LottoStoreTest.kt

```diff
@@ -16,14 +18,54 @@ internal class LottoStoreTest {
         assertThat(result.size).isEqualTo(ticketCount)
     }
 
+    @ParameterizedTest
+    @MethodSource("buyLottoNumbersTicketsProvider")
+    fun `수동으로 사고싶은 만큼 LottoNumbers를 주면 알맞은 LottoTicket을 준다`(
+        money: Money,
+        listOfLottoNumbers: List<LottoNumbers>,
+        lottoTickets: LottoTickets
+    ) {
+        val result = store.buy(money, listOfLottoNumbers)
+
+        assertThat(result.size).isEqualTo(lottoTickets.size)
+        assertThat(result[0].candidateNumbers).isEqualTo(lottoTickets[0].candidateNumbers)
+        assertThat(result[1].candidateNumbers).isEqualTo(lottoTickets[1].candidateNumbers)
+    }
+
     companion object {
         @JvmStatic
         fun buyTicketCountProvider(): List<Arguments> {
             return listOf(
-                Arguments { arrayOf(Money.thousand, 1) },
+                Arguments { arrayOf(Money.THOUSAND, 1) },
                 Arguments { arrayOf(Money(3000), 3) },
                 Arguments { arrayOf(Money(2500), 2) },
-                Arguments { arrayOf(Money.zero, 0) }
+                Arguments { arrayOf(Money.ZERO, 0) }
+            )
+        }
+
+        @JvmStatic
+        fun buyLottoNumbersTicketsProvider(): List<Arguments> {
+            val listOfLottoNumbers = listOf(
+                LottoNumbersFactory.create(listOf(1, 2, 3, 4, 5, 6)),
+                LottoNumbersFactory.create(listOf(7, 8, 9, 10, 11, 12))
```

Comment:
생성자의 수가 늘어날수록 클래스를 사용하기 위한 유연성은 높아집니다.

```suggestion
                LottoNumbers(1, 2, 3, 4, 5, 6),
                LottoNumbers(7, 8, 9, 10, 11, 12)
```